### DISCUSSION TITLE
v4 upgrade docs confusion (emailVerified vs email_verified)

### DISCUSSION THREAD

**Discussion by TheUnlocked** [nextauthjs/next-auth#2839] [Help] [answered]
In the [v4 schema changes documentation](https://next-auth.js.org/getting-started/upgrade-v4#schema-changes), it says that `emailVerified` has been renamed to `email_verified`. However, when using the v4 beta, that doesn't seem to be the case (e.g., this code in `callback-handler.ts`: https://github.com/nextauthjs/next-auth/blob/beta/src/server/lib/callback-handler.ts#L198). Is this a typo in the documentation, or is this going to be changed in the code by the time v4 releases?

**Comment by balazsorban44** [comment]
Sorry, that seems to be wrong in the docs! `emailVerified` is the correct naming.

The ones using `snake_case` are coming from a third party library, and used mostly only for the OAuth specific cases, in which `access_token` and others are usually referred to with snake_case in the OAuth specifications as well.

**Comment by balazsorban44** [comment, marked as answer]
Created a PR to fix this! https://github.com/nextauthjs/docs/pull/58


and now merged!
